extensions/influxdb/deployment/src/main/java/org/apache/camel/quarkus/component/influxdb/deployment/InfluxdbProcessor.java

```diff
@@ -16,32 +16,27 @@
  */
 package org.apache.camel.quarkus.component.influxdb.deployment;
 
+import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
-import io.quarkus.deployment.annotations.ExecutionTime;
-import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.ExtensionSslNativeSupportBuildItem;
 import io.quarkus.deployment.builditem.FeatureBuildItem;
-import io.quarkus.deployment.pkg.steps.NativeBuild;
-import org.apache.camel.quarkus.core.JvmOnlyRecorder;
-import org.jboss.logging.Logger;
+import io.quarkus.deployment.builditem.nativeimage.NativeImageProxyDefinitionBuildItem;
 
 class InfluxdbProcessor {
-    private static final Logger LOG = Logger.getLogger(InfluxdbProcessor.class);
 
     private static final String FEATURE = "camel-influxdb";
 
     @BuildStep
-    FeatureBuildItem feature() {
+    FeatureBuildItem feature(BuildProducer<ExtensionSslNativeSupportBuildItem> extensionSslNativeSupport) {
+
+        // Indicates that this extension would like the SSL support to be enabled
+        extensionSslNativeSupport.produce(new ExtensionSslNativeSupportBuildItem(FEATURE));
+
```

Comment:
I think `extensionSslNativeSupport.produce(new ExtensionSslNativeSupportBuildItem(FEATURE));` deserves a separate `@BuildStep` method. The code would be easier to follow that way. 

@lburgazzoli would even say that all `*Native*BuildItem` s should be produced in a separate `InfluxdbNativeProcessor` class. I do not find it necessary as long as there is a just a few methods here.

Reply:
I'm not so strict, you have some freedom :)